packages/nx/src/command-line/explain.ts

```diff
@@ -0,0 +1,94 @@
+import * as chalk from 'chalk';
+import { join, relative } from 'path';
+import { nxConfigExplainer } from '../explain/config-explainers/nx-json';
+import { workspaceRoot } from '../utils/app-root';
+import { readJsonFile } from '../utils/fileutils';
+import { output } from '../utils/output';
+
+export interface ExplainArgs {
+  config: string;
+}
+
+export const defaultNxJsonPath = join(workspaceRoot, 'nx.json');
+
+export function explainHandler(args: ExplainArgs) {
+  if (args.config === 'nx.json') {
+    args.config = defaultNxJsonPath;
+  }
+  const resolvedPath = require.resolve(args.config, {
+    paths: [process.cwd()],
+  });
+  const nxJson = readJsonFile(resolvedPath);
+
+  output.log({
+    title:
+      resolvedPath === defaultNxJsonPath
```

Comment:
This should always say 

`Explanation of settings in ${absolutePath}`

Reply:
I can remove the special case for a workspace's own nx.json config if you feel strongly about it, but I intentionally did not go for an absolute path and instead added the CWD relative path formatting in the output because the full absolute path is very verbose, and doesn't yield any advantages over a CWD relative path when it comes to in-terminal navigation (e.g. being able to CMD+click):

**Absolute** 👎 
![image](https://user-images.githubusercontent.com/900523/166203559-0fc7f34d-58cd-46bb-93b9-7e1e677ee228.png)

**CWD relative per PR** 👍 
![image](https://user-images.githubusercontent.com/900523/166203644-8a647597-8473-406f-b7d5-eef4ef568c9b.png)

Reply:
Sorry I realised am making assumptions about what your concerns were with not having the absolute path (the clarity and navigability), but please let me know if that assumption was incorrect!